packages/editor/src/components/entities-saved-states/index.js

```diff
@@ -36,56 +27,31 @@ function identity( values ) {
 	return values;
 }
 
-function isPreviewingTheme() {
+export default function EntitiesSavedStates( { close } ) {
+	const isDirtyProps = useIsDirty();
 	return (
-		window?.__experimentalEnableThemePreviews &&
-		getQueryArg( window.location.href, 'theme_preview' ) !== undefined
+		<EntitiesSavedStatesExtensible
+			{ ...{
+				...isDirtyProps,
+				close,
+			} }
+		/>
 	);
 }
 
-function currentlyPreviewingTheme() {
-	if ( isPreviewingTheme() ) {
-		return getQueryArg( window.location.href, 'theme_preview' );
-	}
-	return null;
-}
-
-export default function EntitiesSavedStates( { close, onSave = identity } ) {
+export function EntitiesSavedStatesExtensible( {
```

Comment:
We may want to make this version private for now. That will allow us to change the props without having to maintain backwards compatibility. There's an example [here](https://github.com/WordPress/gutenberg/pull/49137/files), but I'm also happy to help with this if you need it.

Reply:
Addressed it here! Thank you!
https://github.com/WordPress/gutenberg/pull/50863/commits/023e17bf910e279a7205013c6cd84c460b9cc3b0